### PR TITLE
[docs] fix document date-time-functions typo

### PR DESCRIPTION
**Pull request by steadyBoy**
1. en: add convert_tz.md
2. en: fix curdate.md, the fix is to modify the title
3. en: add curtime.md
4. en: fix str_to_date.md, the fix is to modify the title
5. zh-CN: fix convert_tz.md, the fix is to modify the description
6. zh-CN: fix curdate.md, the fix is to modify the title

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (I Don't know)
2. Has unit tests been added: (No Need)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by hf200012 — DISMISSED** [review]
LGTM

File: docs/en/sql-reference/sql-functions/aggregate-functions/group_concat.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+{
+    "title": "group_concat",
```
Comment (qidaye) [inline docs/en/sql-reference/sql-functions/aggregate-functions/group_concat.md:3]: `group_concat` already exists in string-functions subdirectory.

File: docs/zh-CN/sql-reference/sql-functions/aggregate-functions/group_concat.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+{
+    "title": "group_concat",
```
Comment (qidaye) [inline docs/zh-CN/sql-reference/sql-functions/aggregate-functions/group_concat.md:3]: Same as above.

File: docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md [hunk 3]
```diff
@@ -25,14 +25,12 @@ under the License.
 -->
 
 # convert_tz
-
+## description
 ## Syntax
```
Comment (qidaye) [inline docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md:29]:
```suggestion
### Syntax
```

**Review by qidaye — DISMISSED** [review]
LGTM

File: docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md [hunk 4]
```diff
@@ -25,14 +25,12 @@ under the License.
 -->
 
 # convert_tz
-
-## Syntax
+## description
+### Syntax
```
Comment (yangzhg) [inline docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md:29]: why？

**Review by yangzhg — COMMENTED** [review]
You should change `docs/.vuepress/sidebar` to make you rename visible

File: docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md (hunk 4 above)
Comment (steadyBoy) [reply docs/zh-CN/sql-reference/sql-functions/date-time-functions/convert_tz.md:29]: Thanks for reminding. It has been modified

File: docs/en/sql-reference/sql-functions/date-time-functions/curdate.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# curdate
+# curdate,current_date
```
Comment (qidaye) [inline docs/en/sql-reference/sql-functions/date-time-functions/curdate.md:27]: Add `current_date` to  keyword too.

File: docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md [hunk 6]
```diff
@@ -24,7 +24,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# curdate
+# curdate,current_date
```
Comment (qidaye) [inline docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md:27]: Same as above.

File: docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md [hunk 7]
```diff
@@ -24,7 +24,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# curdate
+# curdate,`current_date`
```
Comment (qidaye) [inline docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md:27]:
```suggestion
# curdate,current_date
```

Comment (steadyBoy) [reply docs/zh-CN/sql-reference/sql-functions/date-time-functions/curdate.md]: Thanks for reminding. It has been modified

**Review by qidaye — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
hi @steadyBoy ,please rebase the master to solve the conflict.

**Comment by qidaye** [comment]
@steadyBoy your rebase is not correct, please recheck.

**Review by qidaye — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**